lab-nicole/__test__/auth-route.test.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const request = require('superagent');
+const User = require('../model/user.js');
+const serverToggle = require('../lib/server-toggle.js');
+const server = require('../server.js');
+
+require('jest');
+
+const url = 'http://localhost:3000';
```

Comment:
If I used a different PORT variable none of your tests would work because this is hardcoded in